### ISSUE TITLE
Use `<For />` in `InfiniteScrollArea` documentation

### ISSUE THREAD

**Issue by taroj1205** [yamada-ui/yamada-ui#3761]
### Provide the exact quote of the error or issue
We should use `<For />` component when repeating a component
e.g.
```tsx
<For each={items}>{(item, index) => <Text key={index} fontWeight="bold">{item}</Text>}</For>
```

https://yamada-ui.com/components/data display/infinite-scroll-area

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

**Comment by hirotomoyamada** [comment]
@ve1997 

Hi, thanks for the comment!

Yes, pls!

I'll assign to you this issue.

**Comment by ve1997** [comment]
@hirotomoyamada Thank you!
This is my first time contributing to OSS, so I’d like to ask a few questions.

For this contribution, which is a docs fix, is it necessary to add tests or run changeset?
From what I saw in PRs like `FileInput`(https://github.com/yamada-ui/yamada-ui/pull/3850) that have already been merged, it seems unnecessary, but it was mentioned in your article, so I wanted to double-check.

https://zenn.dev/hirotomoyamada/articles/711e4727d7a56e

By the way, thank you for the good article!
